lib/msal-browser/src/config/Configuration.ts

```diff
@@ -316,5 +323,12 @@ export function buildConfiguration(
         system: { ...DEFAULT_BROWSER_SYSTEM_OPTIONS, ...providedSystemOptions },
         telemetry: { ...DEFAULT_TELEMETRY_OPTIONS, ...userInputTelemetry },
     };
+
+    // Throw an error if user has set OIDCOptions without being in OIDC protocol mode
+    if(overlayedConfig.auth.protocolMode !== ProtocolMode.OIDC && 
```

Comment:
If you move this up a little bit and use the user provided values you can build the behavior I mentioned above

Reply:
I get it now, I can change that